discovery-provider/src/tasks/index.py

```diff
@@ -281,7 +299,7 @@ def fetch_ipfs_metadata(
                 track_metadata_format
                 if cid_type[cid] == "track"
                 else user_metadata_format,
-                None,
+                user_to_cnode_endpoints[cid_to_user[cid]],
```

Comment:
can we define this as a named var outside? 
```python
if cid_type[cid] == "track":
    # assign args
else:
    cnode_endpoint = user_to_cnode_endpoints[cid_to_user[cid]]
```

Reply:
i pulled these out into named variables